server/scripts/create_database.sql

```diff
@@ -24,10 +24,17 @@ CREATE TABLE clients
     gender VARCHAR(120),
     date_of_birth DATE,
     postcode VARCHAR(20),
-    referring_agency VARCHAR(120),
-    pick_up_date TIMESTAMP,
-    is_declined BOOLEAN NOT NULL DEFAULT FALSE,
-    UNIQUE (phone_number)
+    referring_agency VARCHAR(120)
+    uuid CHAR(36) UNIQUE
```

Comment:
It looks like we've removed the `pick_up_date` and `is_declined` columns—shall we add them back in?